### PR TITLE
[MonologBundle] Added (commented) configuration for log format

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: symfony/monolog-bundle/3.1/config/packages/dev/monolog.yaml [hunk 1]
```diff
@@ -17,3 +17,7 @@ monolog:
             type: console
             process_psr_3_messages: false
             channels: ["!event", "!doctrine", "!console"]
+            # # As of Symfony 4.3 you can configure the log output:
```
Comment (Pierstoval) [inline symfony/monolog-bundle/3.1/config/packages/dev/monolog.yaml]: Is there a documentation link that explains all format options?

Comment (lyrixx) [reply symfony/monolog-bundle/3.1/config/packages/dev/monolog.yaml]: I can not find it on symfony.com.
But here is the list https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Monolog/Formatter/ConsoleFormatter.php#L50-L54

Would you mind create a PR? :heart:

Comment (Pierstoval) [reply symfony/monolog-bundle/3.1/config/packages/dev/monolog.yaml]: I don't really have time for this right now, and I guess your recipe would be merged before I could have time to work on it 🤣 , but I think it's an easy-pick so I proposed it in symfony/symfony-docs#11144 to see if someone wants an easy first contribution 😄

**Comment by Nyholm** [comment]
I would much rather have a link to the documentation that showing comments like this in the recipes.

**Comment by lyrixx** [comment]
@Nyholm Let's merge it?

**Pull request merged**